source/_templates/layout.html

```diff
@@ -7,11 +7,20 @@
 <!DOCTYPE html>
 {%- endblock %}
 
-{# Extra files needed in head #}
+{# Extra tags needed in head #}
 {% block extrahead %}
   <link rel="icon" href="{{ pathto('_static/favicon.ico', 1) }}" type="image/png" />
   <link rel="stylesheet" href="{{ pathto('_static/bootstrap.min.css', 1) }}" media="screen" />
   <link rel="stylesheet" href="{{ pathto('_static/normalize.css', 1) }}" media="screen">
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-86671044-2', 'auto');
+    ga('send', 'pageview');
```

Comment:
From [this page](https://developers.google.com/analytics/devguides/collection/analyticsjs/pages) it looks like the default snippet I used will track individual page views. Let's try it out and monitor traffic over the next couple days.